src/core/ext/transport/cronet/transport/cronet_transport.c

```diff
@@ -155,7 +156,9 @@ struct op_state {
   bool flush_read;
   bool flush_cronet_when_ready;
   bool pending_write_for_trailer;
-  bool unprocessed_send_message;
+  bool pending_send_message;
+  bool pending_recv_trailing_metadata;
```

Comment:
add "// User has requested trailing metadata or status". Also explain the condition for flushing.